lib/plugins/aws/package/compile/events/alb/lib/targetGroups.test.js

```diff
@@ -13,6 +13,8 @@ describe('#compileTargetGroups()', () => {
     serverless.setProvider('aws', new AwsProvider(serverless));
     serverless.service.service = 'some-service';
     serverless.service.provider.compiledCloudFormationTemplate = { Resources: {} };
+    serverless.service.functions.first = {};
+    serverless.service.functions.second = {};
```

Comment:
I had to add those lines to prevent the destructuring assignment (`const { provisionedConcurrency } = this.serverless.service.functions[functionName];`) in lib/plugins/aws/package/compile/events/alb/lib/target.js from failing.